content/content-bundles/resources/content.properties

```
@@ -445,6 +445,8 @@ setpermis = Set permissions for resources in folder
 
 setpermis1 = Set permissions for resources in worksite 
 
+permissions.warning = Removing the "Read resources" permission may prevent other tools from functioning correctly, and taking it away from any role is not recommended.
```

Removing the "Read resources" permission will prevent the role from viewing or uploading files and may prevent other tools from functioning correctly. To hide specific files, please use "Edit Details" to control access.

---

The "to control access" part seems redundant to the "To hide specific files" part. Instead, I recommend mentioning where the "Edit Details" is because it took me a moment to think where you were suggesting. 

```suggestion
permissions.warning = Removing the "Read resources" permission will prevent the role from viewing or uploading files and may prevent other tools from functioning correctly. To hide specific files, use "Edit Details" in the Actions menu.
```